### PR TITLE
Added new course

### LINKED ISSUE #2140: [Add] Code with Harry
### Site URL

https://www.codewithharry.com/videos/web-dev-using-flask-and-python-1/

### Description of the above provided site

Web development using Flask and Python
It's a Youtube course helps students learn in an interactive way
It has many modules which help students learn step by step

### Category

Frontend

### Sub Category

Python

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by dalvishruti14**
Closes #2140

### CODE REVIEW

File: database/youtube/web-development.json [hunk 1]
```diff
@@ -31,6 +31,14 @@
     "subcategory": "web-development",
     "language": "english"
   },
+  {
+    "name": "CodeWithHarry",
+    "description": "This course will teach you everything you need to know about web development using Flask. It will lead you through each and every concept of Flask and will give you a good initial start.",
+    "url": "https://www.codewithharry.com/videos/web-dev-using-flask-and-python-1/",
+    "category": "youtube",
+    "subcategory": "web-development",
+    "language": "Hindi"
```
Comment (CBID2) [inline database/youtube/web-development.json:40]:
```suggestion
    "language": "hindi"
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Looks good @dalvishruti14! :) Just needs some improvement.

**Review by CBID2 — APPROVED** [review]
Looks great to me @dalvishruti14

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Comment by dalvishruti14** [comment]
@rupali-codes can you please review it

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**